src/main/java/leduc/command/DeadlineCommand.java

```diff
@@ -55,14 +57,31 @@ public void execute(TaskList tasks, Ui ui, Storage storage)
             throw new EmptyDeadlineDateException();
         } else {
             String description = taskDescription[0].trim();
-            String deadlineString = taskDescription[1].trim();
+            String[] prioritySplit = taskDescription[1].trim().split("prio");
+            String deadlineString = prioritySplit[0].trim();
             //date format used: dd/MM/yyyy HH:mm
             if (deadlineString.isBlank()) {
                 throw new EmptyDeadlineDateException();
             }
             else {
                 Date d = new Date(deadlineString);
-                DeadlinesTask newTask = new DeadlinesTask(description, d);
+                DeadlinesTask newTask = null;
+                if (prioritySplit.length == 1){
+                    newTask = new DeadlinesTask(description, d);
+                }
+                else {
+                    int priority = -1 ;
+                    try{
+                        priority = Integer.parseInt(prioritySplit[1].trim());
+                    }
+                    catch(Exception e){
+                        throw new PrioritizeLimitException();
+                    }
+                    if (priority < 0 || priority > 9) {
+                        throw new PrioritizeLimitException();
+                    }
+                    newTask = new DeadlinesTask(description,"[✗]",d,priority);
```

Comment:
Why did you specify [✗]

Reply:
These constructors for different Tasks are already used in storage. It is to avoid to create an other constructor or to modify the first constructor and I will have to create a task by input the standard priority 5 in each instantiation in hard.

Do you think it is better to create another constructor or modify the first one ?